jellyfin-core/src/main/kotlin/org/jellyfin/sdk/discovery/RecommendedServerDiscovery.kt

```diff
@@ -27,103 +31,101 @@ public class RecommendedServerDiscovery(
 		val responseTime: Long,
 	)
 
-	private suspend fun getSystemInfoResult(address: String): SystemInfoResult? {
+	private val logger = LoggerFactory.getLogger("RecommendedServerDiscovery")
+
+	@Suppress("MagicNumber")
+	private fun assignScore(result: SystemInfoResult, parentResult: SystemInfoResult? = null): RecommendedServerInfo {
```

Comment:
What's that parent meant for?

Reply:
- Remove includeAppendedServers: too complex, not worth the amount of additional requests

if only I removed it everywhere...

Reply:
Ah ok, well then.